ROLES.md

```diff
@@ -83,6 +84,17 @@ Here is the set of roles we use within the Istio community, the general responsi
     <td>n/a</td>
     <td>‘approver’ entry in one or more OWNERS files within the istio/contrib repo</td>
   </tr>
+
+  <tr>
+    <td><a href="#sponsor">Sponsor</a></td>
```

Comment:
Since you're adding a sponsor role, can you refer to it from places we mention being sponsored by members? (e.g. under member, collaborator).

Also maybe use different language where we say sponsored by TOC to distinguish sponsorship?

Reply:
Done.

The TOC now appoints rather than sponsors.